pkg/metricsevent/storage/metrics.go

```diff
@@ -47,13 +50,65 @@ var (
 			Help:      "Total number of sdk timeout errors",
 		}, []string{"tag"})
 
+	// TODO: Remove this event after grpc server is removed.
 	sdkInternalErrorCounter = prometheus.NewCounterVec(
 		prometheus.CounterOpts{
 			Namespace: "bucketeer",
 			Subsystem: "metrics_event",
 			Name:      "sdk_internal_error_total",
 			Help:      "Total number of sdk internal errors",
 		}, []string{"tag"})
+
+	// We added prefix "sdk_api" because of avoiding name collision
+	sdkLatencyHistogram = prometheus.NewHistogramVec(
```

Comment:
@masaaania 

This will be helpful to create the dashboard in grafana.

Reply:
Thanks, they almost comply with [the conventions](https://prometheus.io/docs/practices/naming/) 👍 
I think it might be better to use `sdk_api_bytes` instead of `sdk_api_size`.

Reply:
Thx. I think it'll be better if we use sdk_api_byte_size.

Reply:
The conventions set "A metric name should have a suffix describing the unit, in plural form. "
https://prometheus.io/docs/practices/naming/#metric-names

But, if we have a common understanding, the suffix unit can be omitted, and `sdk_api_reponse_size` is good.